### ISSUE TITLE
About add other loss to the bootstrapped cross entropy

### ISSUE THREAD

**Issue by zhouweii234** [hkchengrex/STCN#79]
May I ask you how to do the scaling as mentioned in this answer? https://github.com/hkchengrex/STCN/issues/63#issuecomment-928091282

The bootstrapped cross entropy is about 0.04-0.06 before 50k steps, after that is about 0.1. So should I keep the added loss between 0.04-0.06? Or I should keep it around 0.1?

**Comment by zhouweii234** [comment]
Can you please give me some guidance about how to monitor the gradient? It seems that Keras can use `write_grads` flag, and tensorflow can use `log_histograms` flag. But I didn't find any information about pytorch. Does pytorch have similar flags? I still need to implement this in code by myself?
